omnibus/package-scripts/updater-deb/postinst

```diff
@@ -81,11 +80,11 @@ fi
 # Set proper rights to the dd-agent user
 chown -R dd-agent:dd-agent ${CONFIG_DIR}
 chown -R dd-agent:dd-agent ${LOG_DIR}
-chown -R dd-updater:dd-updater ${INSTALL_DIR}
-chown -R dd-updater:dd-updater ${PACKAGES_DIR}
-
-chmod -R 755 ${INSTALL_DIR}
+chown -R dd-agent:dd-agent ${INSTALL_DIR}
+chown -R dd-agent:dd-agent ${PACKAGES_DIR}
+chown -R dd-agent:dd-agent ${PACKAGES_LOCK_DIR}
 chmod -R 755 ${PACKAGES_DIR}
+chown -R 666 ${PACKAGES_LOCK_DIR}
```

Comment:
Are you sure we need the lock dir to be world writable?

Reply:
Yes, this can be written by customer applications in the tracer case.
When a tracer is injected by a process and we write the PID at injection, the write will be with the customer process user

Reply:
I added a comment to clarify

Reply:
Ah ok, much clearer! Thanks